tests/test_adopt.py

```diff
@@ -80,11 +80,19 @@ def test_new_module(extra_args, tmp_path: Path):
     )
     assert (module / "src" / "my_module").is_dir()
     assert check_output("git", "branch", cwd=module).strip() == "* main"
+
     check_output("python", "-m", "venv", "venv", cwd=module)
-    check_output("venv/bin/pip", "install", "--upgrade", "pip", cwd=module)
-    check_output("venv/bin/pip", "install", ".[dev]", cwd=module)
+
+    python_exec = Path("venv") / "bin" / "python"
+    if name == "nt":
+        python_exec = (module / "venv" / "Scripts" / "python.exe").absolute()
+    else:
+        check_output(
+            str(python_exec), "-m", "pip", "install", "--upgrade", "pip", cwd=module
+        )
+    check_output(str(python_exec), "-m", "pip", "install", ".[dev]", cwd=module)
```

Comment:
Did you mean to only run the upgrade of pip on non-windows? I was expecting something like:
```suggestion
    if name == "nt":
        python_exec = (module / "venv" / "Scripts" / "python.exe").absolute()
    else:
        python_exec = Path("venv") / "bin" / "python"
    check_output(
        str(python_exec), "-m", "pip", "install", "--upgrade", "pip", cwd=module
    )
    check_output(str(python_exec), "-m", "pip", "install", ".[dev]", cwd=module)
```

Reply:
Huh this seems to work... Back when I tried this I was getting an error but I guess it was something else that I fixed since.